src/broker/comm.bif

```diff
@@ -145,3 +145,45 @@ function Broker::__node_id%(%): string
 	zeek::Broker::Manager::ScriptScopeGuard ssg;
 	return zeek::make_intrusive<zeek::StringVal>(broker_mgr->NodeID());
 	%}
+
+function Broker::__set_metrics_export_interval%(value: interval%): bool
+	%{
+	// This BIF may run prior to broker::Manager::InitPostScript. In this case,
+	// broker_mgr is still null but we can safely ignore this event because the
+	// Manager is going to initialize Broker using the most recent value of the
+	// corresponding option.
+	zeek::Broker::Manager::ScriptScopeGuard ssg;
+	if ( broker_mgr )
+		broker_mgr->SetMetricsExportInterval(value);
+	return zeek::val_mgr->True();
+	%}
+
+function Broker::__set_metrics_export_topic%(value: string%): bool
+	%{
+	zeek::Broker::Manager::ScriptScopeGuard ssg;
+	if ( broker_mgr )
+		broker_mgr->SetMetricsExportTopic(value->ToStdString());
+	return zeek::val_mgr->True();
+	%}
+
+function Broker::__set_metrics_export_endpoint_name%(value: string%): bool
+	%{
+	zeek::Broker::Manager::ScriptScopeGuard ssg;
+	if ( broker_mgr )
+		broker_mgr->SetMetricsExportEndpointName(value->ToStdString());
+	return zeek::val_mgr->True();
+	%}
+
+function Broker::__set_metrics_export_prefixes%(filter: string_vec%): bool
+	%{
+	zeek::Broker::Manager::ScriptScopeGuard ssg;
+	if ( broker_mgr )
+	{
+		std::vector<std::string> slist;
+		auto* vval = filter->AsVectorVal();
+		for ( unsigned index = 0; index < vval->Size(); ++index )
+			slist.emplace_back(vval->StringValAt(index)->ToStdString());
+		broker_mgr->SetMetricsExportPrefixes(std::move(slist));
+	}
```

Comment:
Nitpick: missing a tab-indent on the braces.

Reply:
Yep, indented properly now.